jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/BaseImageAssert.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.jdi.light.ui.bootstrap.asserts;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.ui.bootstrap.elements.common.BaseImage;
+import org.hamcrest.Matcher;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+import static org.hamcrest.Matchers.is;
+
+public abstract class BaseImageAssert<E extends BaseImage> extends UIAssert<BaseImageAssert, BaseImage> {
+
+    @JDIAction("Assert that '{name}' height {0}")
+    public BaseImageAssert height(Matcher<Integer> condition) {
+        jdiAssert(element.height(), condition);
+        return this;
+    }
+    public BaseImageAssert height(int height) { return height(is(height)); }
```

Comment:
JDIAction?

Reply:
Could you add details to the note, please?